components/organisms/Video/volume.ts

```diff
@@ -0,0 +1,67 @@
+import { VideoJsPlayer } from "video.js";
+import VimeoPlayer from "@vimeo/player";
+
+type Volume = {
+  /** 音量 0-1 */
+  volume: number;
+  /** ミュート */
+  muted: boolean;
+};
+
+const Key = "volumePersisterVolume";
+const MuteKey = "volumePersisterMute";
+const Muted = "true";
+
+function save({ volume, muted }: Volume) {
+  localStorage.setItem(Key, volume.toString());
+  if (muted) localStorage.setItem(MuteKey, Muted);
+  else localStorage.removeItem(MuteKey);
+}
+
+function load(): Volume | undefined {
+  const volume = localStorage.getItem(Key);
+  if (volume == null) return;
+  const muted = localStorage.getItem(MuteKey) === Muted;
+  return {
+    volume: Number(volume),
+    muted,
+  };
+}
+
+async function intoVimeo(player: VimeoPlayer) {
+  player.on("volumechange", async function (event: { volume: number }) {
+    // @ts-expect-error
+    const muted = await (player.getMuted() as Promise<boolean>);
```

Comment:
```suggestion
    const muted = await player.getMuted();
```
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48347 にて対応済み。 おそらく `@types/vimeo__player@2.10.0` にアップデートしてもらえれば `as Promise<boolean>`  は省略できるハズ

Reply:
Video関連は本当にVideoPlayerのものをコピペしただけの状態で、これから触っていく予定です

そしてコメントいただけて触りやすくなりそうでありがたいです…！